feature/stamps/src/main/java/io/github/droidkaigi/confsched2023/stamps/StampsScreen.kt

```diff
@@ -69,15 +72,19 @@ private fun StampsScreen(
         modifier = Modifier.testTag(StampsScreenTestTag),
         snackbarHost = { SnackbarHost(snackbarHostState) },
         content = { padding ->
-            Column(
-                Modifier
-                    .padding(padding),
-            ) {
-                Text(
-                    text = "Please implement StampsScreen!!!",
-                    style = MaterialTheme.typography.titleLarge,
-                )
-            }
+            val layoutDirection = LocalLayoutDirection.current
+
+            StampsSheet(
+                modifier = Modifier.padding(
+                    PaddingValues(
```

Comment:
Maybe we don't have to use PaddingValues and you can use "Modifier.padding(top =xxx, start=xxx" but we can make a issue for this so it's up to you if we do it or not. 👍

Reply:
👀

Reply:
Android Lint prints an error when ignoring the padding from Scaffold by the way.

Reply:
@takahirom 
Thank you for your comment.
As I read it, I felt that PaddingValues is indeed redundant.
So I thought it would be better to do something like this:

```
modifier = Modifier.padding(
    top = padding.calculateTopPadding() + 20.dp,
    start = padding.calculateStartPadding(layoutDirection) + 16.dp,
    end = padding.calculateEndPadding(layoutDirection) + 16.dp,
)
```

However, the point to note here is that the bottom padding is not specified.
Specifying bottom padding causes unnecessary space between BottomNavigation and Screen.

Reply:
Oh, I misunderstood the first comment's intention. Yes, instantiating PaddingValues is truly redundant. Sorry for confusing you all!

Reply:
So maybe all you have to do is remove PaddingValues and leave it as it is for other codes 👀